src/infrastructure/MetadataHttp.ts

```diff
@@ -259,7 +231,7 @@ export class MetadataHttp extends Http implements MetadataRepository {
                 metadataEntry.targetPublicKey,
                 UInt64.fromHex(metadataEntry.scopedMetadataKey),
                 metadataEntry.metadataType.valueOf(),
-                metadataEntry.valueSize,
+                Convert.decodeHex(metadataEntry.value).length,
```

Comment:
Should we just remove the size from the transaction model? I took it out in Java

Reply:
what is the reasoning behind removing it ?

Reply:
this size is not defined in OpenAPI, and I believe because it is not returning from the rest.

Reply:
It can be easily calculated from `value.length` I think. as how the SDK is doing here